LayoutTests/css3/filters/effect-drop-shadow-clip-abspos.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<style>
+.container {
+	    filter: drop-shadow(5px 5px 5px black);
+      transition: filter 2s ease;
+	}
+.circle-mask {
+	    border-radius: 80px;
+	    overflow: hidden;
+	    width: 100px;
+	    height: 100px;
+	    position: absolute;
+        filter: blur(20px);
+	}
+	.green-box {
+	    width: 100px;
+	    height: 100px;
+	    background-color: green;
+	}
+	</style>
+	<div class="container">
+	  <div class="circle-mask">
+	    <div class="green-box"></div>
+</div>
+</div>
```

Comment:
small nit: can you re-indent your test + reference in a readable way? (our standard is usually 4 spaces)

Reply:
Will do once back at home. 👍

Reply:
Done.